inc/class-s3-uploads.php

```diff
@@ -237,6 +238,22 @@ public function wp_filter_read_image_metadata( $meta, $file ) {
 		return $meta;
 	}
 
+	/**
+	 * Add the DNS address for the S3 Bucket to list for DNS prefetch.
+	 *
+	 * @param $hints
+	 * @param $relation_type
+	 *
+	 * @return array
+	 */
+	function wp_filter_resource_hints( $hints, $relation_type ) {
+		if ( 'dns-prefetch' === $relation_type ) {
```

Comment:
This could use the `if ( !== ) { return $hints }` pattern but I think it's clearer and less error prone this way.

Values and names checked, all correct.

Reply:
I did think about doing it this way, but personally I thought this way was easier to read. Willing to change it if anyone disagrees...